### ISSUE TITLE
[Feature] Command to output nicely formatted TXT record contents.

### ISSUE THREAD

**Issue by Jieiku** [rspamd/rspamd#4429]
## Summary

When you generate a DKIM record, its not formatted very good for putting the txt info into a TXT record at your registrar. Would be nice to have a command to do this.

## Motivation

I wrote a little script, which handles it for me, but I think a native command would be better. I know for a fact I am not the only one to run into this, countless guides on the internet talk about formatting the output correctly for the registrars.

## Describe alternatives you've considered

Currently I do the following:

```bash
sudo rspamadm dkim_keygen -b 4096 -s mail -k /var/lib/rspamd/dkim/mail.key | sudo tee -a /var/lib/rspamd/dkim/mail.pub
sudo chown -R _rspamd: /var/lib/rspamd/dkim
sudo chmod 440 /var/lib/rspamd/dkim/mail.{key,pub}
echo -e "\n\033[1;93mPaste this output directly into your DNS registrars TXT dns record mail._domainkey\033[0m"
sudo sed -e ':a;N;$!ba;s/\n/ /g; s/\s\+/ /g; s/\" \"//g; s/mail\._domainkey IN TXT ( \"//g; s/\" ) ;//g' /var/lib/rspamd/dkim/mail.pub
```

## Additional context

Would be a really good new feature I think, but until/if it is implemented, I thought the script I wrote might be useful to others.

**Comment by vstakhov** [comment]
So literally you need a public key without splits and without all DNS related labels, right?

**Comment by Jieiku** [comment]
Yes, then when you goto your Registrar, you create the record like so:

- type: `TXT`
- name: `mail._domainkey`
- content: from the previous command but should look similar to this (all one line):
`v=DKIM1; k=rsa; p=MIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEAzE9bfwdmRPFdYYyxuSPgRljGlsOl13XCk1ot5ijKjiJkzaVrttX8cokoS2/OyTe5Fw+JMK4v1EySmfuA7b5K7grgPir5qorXJwQ6ngDm4nEgHXSDQT4pOxtC8EhTk8FpsRZt0pVoGUZaaQHXxFQ5WqvYXEVXON/PwP/xvtaEcTiKyiHT/0zCUB7YbCVjljzpKwsYpU0QDc+ho/N8wuyOxtsl7TtGBeiauf5LqkhBRs/fw5M7M07v61uRTRIdcTS3Gvv/i+EI+tODUPQi374xL2RsEeIIN2Pt1uA34idrVZibRir5WXywDH+M3H4xOjsFc9kuIT9cjtLS6gHXLoEcEURKsxtGnZZ7wIOG0wpxSZDytOE2gAO11AlMvdpe6Grm7BM1ba25RQu9IWCCJuok8GU0tEMBa1y/EqcSa1oYyu/2rsi6+PMtEGSlh+k6matPl0KFw+/qGMBQVZZZylM0Jdax7LTQHQBT0+zXdtOImGjuKxyeQ5rZMayB2Sz7EuhamvcThxSdvVDVCkxGNa0r8jRInx9ski4FTqbmRICRmOaJ7a6W81BLPrVJHtV2cL2YT10c23IBf5eBOSExp5KSB96Aa/N+oAu5X51FTzJxvFJjs3envUdKhWksttInsZIPs8iozGshi2W/aSKOpCe86NiXGu1sOHtjvlrflfBcCbECAwEAAQ==`
